src/explore-education-statistics-frontend/src/modules/table-tool/components/TableToolShare.tsx

```diff
@@ -0,0 +1,118 @@
+import ButtonText from '@common/components/ButtonText';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import UrlContainer from '@common/components/UrlContainer';
+import { TableHeadersConfig } from '@common/modules/table-tool/types/tableHeaders';
+import mapUnmappedTableHeaders from '@common/modules/table-tool/utils/mapUnmappedTableHeaders';
+import permalinkService from '@common/services/permalinkService';
+import {
+  SelectedPublication,
+  TableDataQuery,
+} from '@common/services/tableBuilderService';
+import Link from '@frontend/components/Link';
+import React, { useEffect, useState } from 'react';
+
+interface Props {
+  currentTableHeaders?: TableHeadersConfig;
+  query: TableDataQuery;
+  selectedPublication: SelectedPublication;
+}
+
+const TableToolShare = ({
```

Comment:
Probably a good idea to write some tests for this component :slightly_smiling_face: